docs/ase-file-specs.md

```diff
@@ -202,6 +203,8 @@ entire layers layout:
     STRING      Layer name
     + If layer type = 2
       DWORD     Tileset index
+    UUID        Layer's universally unique identifier (present only if the
+                [main header](#header) "Flags" field has the bit 3 enabled)
```

Comment:
Links don't work in Markdown code blocks:
```suggestion
    + If file header flags have bit 4:
      UUID      Layer's universally unique identifier
```
Here bit 4 is not fourth (or fifth) bit, but value 4, in the User Data Chunk we are already using this notation.